src/scripts/main.js

```diff
@@ -1 +1,9 @@
 'use strict';
+(() => {
+  let template = '';
+  for (let i = 0; i < 31; i++) {
+    template += '<div class="calendar__day"></div>';
+  }
```

Comment:
It's better to use an array and the `join` method for concatenating strings in a loop for performance reasons. Each time you use '+=' to concatenate strings, a new string is created. This can lead to performance issues when dealing with large amounts of data. Here's an example of how you can do it: `let template = Array(31).fill('<div class="calendar__day"></div>').join('');`